src/app/docs/sectioned-form/sectioned-form-docs.component.html

```diff
@@ -0,0 +1,35 @@
+<sky-docs-demo-page
+  moduleName="SkySectionedFormModule"
+  moduleSourceCodePath="src/app/public/modules/sectioned-form/"
+  pageTitle="Sectioned form"
+>
+  <sky-docs-demo-page-summary>
+    The page summary displays critical information and actions for users to access quickly and frequently.
```

Comment:
Need to swap this text out.
```suggestion
    Sectioned forms combine multiple forms and display large amounts of related information.
```